### PR TITLE
Change the way we add tier to the Smart Contract

### PR DESCRIPTION
**Pull request by evgeny-s**
[Ticket](https://cerenetwork.atlassian.net/browse/CBI-899)
* Changed tier configuration
* Fixed tests

### CODE REVIEW

**Comment by naure** [comment]
Is the decoding of the ServiceTier happening in the node RPC or in the JS client-side?
When the DDC calls a node RPC, does it get raw data, or a JSON formatted nicely like that?

**Comment by evgeny-s** [comment]
@naure 
JSON format

File: cere02/lib.rs [hunk 1]
```diff
@@ -199,69 +157,86 @@ mod ddc {
...
 
     impl Ddc {
-        /// Given a tier id: 1, 2, 3
-        /// return the fee required
-        #[ink(message)]
-        pub fn tier_deposit(&self, tid: u64) -> Balance {
-            //self.tid_in_bound(tier_id)?;
-            if tid > 3 {
-                return 0 as Balance;
+        fn calculate_new_tier_id(&self) -> u64 {
```
Comment (naure) [inline cere02/lib.rs:192]: Have you considered using Vec or Stash? This looks similar.

(https://paritytech.github.io/ink/ink_storage/collections/index.html)

**Review by naure — APPROVED** [review]

File: cere02/lib.rs (hunk 1 above)
Comment (evgeny-s) [reply cere02/lib.rs:192]: I wanted the ID started from 1 and increase all the time. Also there is a possibility when some tier removed, or etc, we need still keep increment.

**Pull request merged**